Assets/HoloToolkit/Utilities/Scripts/WorldAnchorManager.cs

```diff
@@ -136,6 +137,36 @@ public void RemoveAnchor(GameObject gameObjectToUnanchor)
         }
 
         /// <summary>
+        /// Removes all anchors from the scene and deletes them from the anchor store.
+        /// </summary>
+        public void RemoveAllAnchors()
+        {
+            // This case is unexpected, but just in case.
+            if (AnchorStore == null)
+            {
+                Debug.LogError("remove all anchors called before anchor store is ready.");
+            }
+
+            WorldAnchor[] anchors = FindObjectsOfType<WorldAnchor>();
```

Comment:
No worries. Yeah I think it would be a good addition, but there's no real good way to prevent users from adding anchors outside of the `WorldAnchorManager` and making sure they get registered appropriately.